src/xrc-tests/src/tests/get_icp_xdr_rate.rs

```diff
@@ -24,38 +24,30 @@ use crate::{
+//           3969300000, 4005000000, 4045454545, 4045454545, 4105263154, 4105263154, 4115789469, 4116842101, 4126315785,
+//           4126315785, 4126500000, 4215789469, ]
 /// 5. The XRC divides the ICP/USD by the forex rate CXDR/USD. The division works by inverting CXDR/USD to USD/CXDR then multiplying
 ///    USD/CXDR and ICP/USD resulting in ICP/CXDR.
 ///     i. For request 1, this results in the following rates:
-///        [ 2829957503, 2829957503, 2829957503, 2829957503, 2837213804, 2837213804, 2837939434, 2837939434,
-///          2844470105, 2844470105, 2844470105, 2844470105, 2860276430, 2860276430, 2860276430, 2860276430,
-///          2867610472, 2867610472, 2868343877, 2868343877, 2874944514, 2874944514, 2874944514, 2874944514,
-///          2906148666, 2906148666, 2917481962, 2917481962, 2917481962, 2917481962, 2924962685, 2924962685,
-///          2925710757, 2925710757, 2932443408, 2932443408, 2932443408, 2932443408, 2937283872, 2937283872,
-///          2947149687, 2947149687, 2947149687, 2947149687, 2947149687, 2947149687, 2947149687, 2947149687,
-///          2954507762, 2954507762, 2954507762, 2954507762, 2955263391, 2955263391, 2955263391, 2955263391,
-///          2962064048, 2962064048, 2962064048, 2962064048, 2962064048, 2962064048, 2962064048, 2962064048,
-///          2996029553, 2996029553, 3026292477, 3026292477, 3026292477, 3026292477, 3071033641, 3071033641,
-///          3071033641, 3071033641, 3078908086, 3078908086, 3079695530, 3079695530, 3086782531, 3086782531,
-///          3086782531, 3086782531, 3153715315, 3153715315, ]
+///        [ 2860468811, 2860468811, 2860468811, 2860468811, 2867803346, 2867803346, 2868536800, 2868536800, 2875137882, 2875137882,
+///          2875137882, 2875137882, 2882472418, 2882472418, 2888501408, 2888501408, 2888501408, 2888501408, 2888501408, 2888501408,
+///          2888501408, 2888501408, 2895907822, 2895907822, 2895907822, 2895907822, 2896648464, 2896648464, 2896648464, 2896648464, 2903314236, 2903314236, 2903314236, 2903314236, 2903314236, 2903314236, 2903314236, 2903314236, 2910720650, 2910720650, 2910720650, 2910720650, 2937481433, 2937481433, 2946854972, 2946854972, 2946854972, 2946854972, 2954411010, 2954411010, 2955166614, 2955166614, 2961967049, 2961967049, 2961967049, 2961967049, 2966268754, 2966268754, 2966268754, 2966268754, 2969523087, 2969523087, 3007915659, 3007915659, 3007915659, 3007915659, 3007915659, 3007915659, 3007915659, 3007915659, 3015628263, 3015628263, 3015628263, 3015628263, 3016399524, 3016399524, 3016399524, 3016399524, 3023340868, 3023340868, 3023340868, 3023340868, 3023340868, 3023340868, 3023340868, 3023340868, 3026193375, 3026193375, 3031053472, 3031053472, 3031053472, 3031053472, 3071240197, 3071240197, 3071240197, 3071240197, 3079115172, 3079115172, 3079902669, 3079902669, 3086990147, 3086990147, 3086990147, 3086990147, 3088898004, 3088898004, 3088898004, 3088898004, 3094865122, 3094865122, 3153927433, 3153927433 ]
```

Comment:
Can we get some better formatting for this? 😄

Reply:
I dont know how that slipped past me. I will fix it